options/m_option.h

```diff
@@ -212,7 +212,7 @@ struct m_sub_options {
     bool (*get_sub_options)(int index, const struct m_sub_options **sub);
 };
 
-#define CONF_TYPE_BOOL          (&m_option_type_bool)
+#define CONF_TYPE_FLAG          (&m_option_type_flag)
```

Comment:
I *think* you could leave the option changes, we just need `CONF_TYPE_FLAG` back for the sake of conversion

Reply:
done